README.md

```diff
@@ -1,10 +1,458 @@
+This command will automatically use the output `wrangler.json` that was included in the build output.
+
+### Next steps
+
+In this tutorial, we created an SPA that could be deployed as a Worker with Workers Assets.
+We then added an API Worker that could be accessed from the front-end code and deployed to Cloudflare.
+Possible next steps include:
+
+- Adding a binding to another Cloudflare service such as a [KV namespace](https://developers.cloudflare.com/kv/) or [D1 database](https://developers.cloudflare.com/d1/)
+- Expanding the API to include additional routes
+- Using a library, such as [tRPC](https://trpc.io/) or [Hono](https://hono.dev/), in your API Worker
+
+## API
+
+### `cloudflare`
+
+The `cloudflare` plugin should be included in the Vite `plugins` array:
+
+```ts
+// vite.config.ts
+
+import { defineConfig } from 'vite'
+import { cloudflare } from '@cloudflare/vite-plugin'
+
+export default defineConfig({
+  plugins: [cloudflare()],
+})
+```
+
+It accepts an optional `PluginConfig` parameter.
+
+### `interface PluginConfig`
+
+- `configPath?: string`
+
+  An optional path to your Worker config file.
+  By default, a `wrangler.toml`, `wrangler.json`, or `wrangler.jsonc` file in the root of your application will be used as the Worker config.
+
+- `viteEnvironment?: { name?: string }`
+
+  Optional Vite environment options.
+  By default, the environment name is the Worker name with `-` characters replaced with `_`.
+  Setting the name here will override this.
+
+- `persistState?: boolean | { path: string }`
+
+  An optional override for state persistence.
+  By default, state is persisted to `.wrangler/state` in a `v3` subdirectory.
+  A custom `path` can be provided or, alternatively, persistence can be disabled by setting the value to `false`.
+
+- `auxiliaryWorkers?: Array<AuxiliaryWorkerConfig>`
+
+  An optional array of auxiliary workers.
+  You can use [service bindings](https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/) to call auxiliary workers from your main (entry) Worker.
+  All requests are routed through your entry Worker.
+  During the build, each Worker is output to a separate subdirectory of `dist`.
+
+> [!NOTE]
+> When running `wrangler deploy`, only your main (entry) Worker will be deployed.
+> If using multiple Workers, each must be deployed individually.
+> You can inspect the `dist` directory and then run `wrangler deploy -c path-to-worker-output-config` for each.
+
+### `interface AuxiliaryWorkerConfig`
+
+- `configPath: string`
+
+  A required path to your Worker config file.
+
+- `viteEnvironment?: { name?: string }`
+
+  Optional Vite environment options.
+  By default, the environment name is the Worker name with `-` characters replaced with `_`.
+  Setting the name here will override this.
+
+## Worker environments
+
+A Worker config file may contain configuration for multiple [environments](https://developers.cloudflare.com/workers/wrangler/environments/).
```

Comment:
```suggestion
A Worker config file may contain configuration for multiple [Wrangler environments](https://developers.cloudflare.com/workers/wrangler/environments/).
```

Reply:
or possibly

```suggestion
A Worker config file may contain configuration for multiple [Cloudflare environments](https://developers.cloudflare.com/workers/wrangler/environments/).
```

Reply:
Oh I see below that you use the term "Worker environment" - so perhaps that would be best here.

Reply:
I've updated it to 'Worker environments'. Let me know if you would rather I use the term 'Cloudflare environments' throughout the readme though. I'm not sure which is better.